docs/book/toc.md

```diff
@@ -16,6 +16,7 @@
   * [Understand stacks](user-guide/starter-guide/understand-stacks.md)
   * [Follow best practices](user-guide/starter-guide/follow-best-practices.md)
   * [Switch to production](user-guide/starter-guide/switch-to-production.md)
+  * [ZenML project templates](user-guide/starter-guide/using-project-templates.md)
```

Comment:
I would move this page over the `Switch to production` page. The switch page is acting as the finishing line of the starter guide and creates a bridge to the deployment guide.

Reply:
Did so